### ISSUE TITLE
Clear description for "trashAssetsBeforeRun"

### ISSUE THREAD

**Issue by ItsWasteD** [cypress-io/cypress-documentation#5618]
### Subject

Clear description for "trashAssetsBeforeRun"

### Description

I already opened a discussion for this topic but I didn't get a response. https://github.com/cypress-io/cypress/discussions/28665

So my question is the following:

> What exactly gets trashed if I set this configuration to true? Is it only image filetypes or everything within the folder specified? I ask this because I use a subfolder of the screenshot folder for a specific command so lets say I have a scheduled run that runs every day that uses "cypress/screenshots" as screenshot folder. Now I also have a run that run twice a day that uses "cypress/screenshots/otherRun" as screenshots folder. So will this folder get deleted every time the daily run executes? Or will only all images be deleted?

**Comment by jennifer-shehane** [comment]
I think technically we move the contents of the video and screenshots folder to trash. These docs could use some better clarity.

**Comment by ItsWasteD** [comment]
Yeah but what folder exactly gets trashed? If I set the the folder to "cypress/screenshots/prod" will only the prod folder get deleted? Is it only the contents? Or will everything inside the screenshots folder get trashed?
